cmd/cpe2cve/cpe2cve.go

```diff
@@ -34,6 +37,33 @@ import (
 	"github.com/facebookincubator/nvdtools/wfn"
 )
 
+// input a
+func processSingle(caches map[string]*cvefeed.Cache, cfg config, cpe string) []string {
+	var cves = []string{}
```

Comment:
A nitpick, but it would be more consistent with the rest of the code to write it as `var cves []string`.